draft-ietf-ohai-ohttp.md

```diff
@@ -456,16 +459,17 @@ Encapsulated Response.
 
 ~~~
 Encapsulated Response {
-  Nonce (Nk),
+  Nonce (max(Nn, Nk)),
```

Comment:
```suggestion
  Nonce (8 * max(Nn, Nk)),
```

Reply:
These are already defined in bits, so we don't need this change.

Reply:
Nn and Nk are bytes, so this would yield the wrong number of bits here, wouldn't it?

Reply:
Sorry, you are right.  I should double check before making bad claims.